ui/src/plugins/dev.perfetto.TraceProcessorTrack/trace_processor_slice_track.ts

```diff
@@ -88,7 +88,9 @@ export async function createTraceProcessorSliceTrack({
       : () => new ThreadSliceDetailsPanel(trace),
     colorizer: (row) => {
       if (row.correlation_id) {
-        return getColorForSlice(row.correlation_id);
+        return getColorForSlice(row.correlation_id, {
+          stripTrailingDigits: false,
```

Comment:
This needs to be configurable. We really need this to be true for Android slices which also use this codepath.

Reply:
Ahhhh I take it back this is on the correlation_id codepath. makes sense this is false here actually.